server/routes/routers/tags.ts

```diff
@@ -76,7 +97,7 @@ tags.post('/:tagId', async (req: any, res: Response) => {
       },
       select: { tags: true },
     });
-    console.log('tag post', updatedUser);
+    // console.log('tag post', updatedUser);
```

Comment:
Remove unused console.log

Reply:
I dont want to remove others console logs if they need them. Since these are patricks, I commented them out so they can be easily added back in